fastlane/lib/fastlane/actions/upload_symbols_to_sentry.rb

```diff
@@ -2,6 +2,11 @@ module Fastlane
   module Actions
     class UploadSymbolsToSentryAction < Action
       def self.run(params)
+
+        UI.important("It's recommended to use the official Sentry Fastlane plugin")
+        UI.important("Github: https://github.com/getsentry/fastlane-plugin-sentry")
+        UI.important("Installation: fastlane add_plugin sentry")
+
```

Comment:
@KrauseFx  it would be great to maybe have this as a system in the zfastlane_corez where we could blacklist a bunch of plugins to show this warning as I would imagine more actions are going to need this warning.